src/js/view/view.js

```diff
@@ -710,6 +713,9 @@ function View(_api, _model) {
             case STATE_IDLE:
             case STATE_ERROR:
             case STATE_COMPLETE:
+                if (_outstream) {
+                    _stopFloating(true);
```

Comment:
@pajong another case where https://github.com/jwplayer/jwplayer/blob/7f093c7fabd2f08e0870543f158998b2172377a7/src/js/view/view.js#L880 will not be pause reason interaction. I'll reference it in the other PR